osquery/tables/system/python_packages.cpp

```diff
@@ -48,8 +48,10 @@ const std::set<std::string> kDarwinPythonPath = {
 };
 // clang-format on
 
-const std::string kWinPythonInstallKey =
-    "SOFTWARE\\Python\\PythonCore\\%\\InstallPath";
+const std::set<std::string> kWinPythonInstallKey = {
+    "SOFTWARE\\Python\\PythonCore\\%\\InstallPath",
+    "SOFTWARE\\Microsoft\\AppModel\\Lookaside\\user\\Software\\Python\\PythonCore\\%\\InstallPath"
```

Comment:
I think 'user' it's your local username on your machine. I suggest replacing it with '%' in order to make it generic.

Reply:
No, it's not. I have no user named "user" on my PC. I have checked this on another machine and the path is the same.